### ISSUE TITLE
Graphs of probability distributions

### ISSUE THREAD

**Issue by endolith** [electology/vse-sim#17]
Currently you show a bunch of symbols of different sizes and colors, with each representing an average of many simulations, correct?  But it's pretty cluttered and hard to see general trends.

Could you just plot the voter happiness directly as a probability distribution for each system? (Does it make sense to refer to the VSE of a single simulation?)

Kind of like http://electology.org/sites/default/files/comparing_voting_methods_simplicity_group_satisfaction.png 

but more like this:

![probdist](https://cloud.githubusercontent.com/assets/58611/22725779/7536688c-ed9d-11e6-935b-d293b007590a.png)

or maybe beanplot style:

![figure-4-region-state-interaction-and-party-politics-notes-bean-plot-with-density-trace](https://cloud.githubusercontent.com/assets/58611/22725785/7d29fd7e-ed9d-11e6-9d23-f97151ba9a83.png)

**Comment by endolith** [comment]
Also, what changed between http://rpubs.com/Jameson-Quinn/vse3 and http://electology.github.io/vse-sim/vse.html ?  You say V321 is better than SRV, but in the older plots it doesn't look like it.

**Comment by jamesonquinn** [comment]
What changed is that in the older graph, strategic voters were using a
different strategy; one that was more likely to backfire and less likely to
work. Backfiring strategies made VSE worse for everyone.

2017-02-14 20:47 GMT-05:00 endolith <notifications@github.com>:

**Comment by jamesonquinn** [comment]
Yes, I could plot distributions like that. (I don't immediately know how,
but I'm sure I could manage it if I needed to.) But there would be separate
distributions for each strategic assumption, and that would make the graph
either less informative or just as cluttered.

2017-02-08 1:26 GMT-05:00 endolith <notifications@github.com>:

**Comment by endolith** [comment]
I'm saying to combine all the strategy mixtures and scenarios into a single distribution for each voting method, without lumping groups of elections together.  I assume that within each scenario and strategy grouping, there are multiple simulations with a spread of results, right?

Do you generate the elections first and then separate out the ones that fit different scenarios, or do you intentionally generate different scenarios directly?

**Comment by jamesonquinn** [comment]
The scenarios are artificially separated. But the strategy mixtures are set
a priori. If I wanted one number for a given method, I would do it by
setting strategy percentage based on strategy effectiveness. This is hard
to do, though, and any actual implementation would involve even several
times more arbitrariness in parameter-setting than what I'm doing so far.

2017-02-15 20:06 GMT-05:00 endolith <notifications@github.com>:

**Comment by endolith** [comment]
(This was my attempt, by the way:  
![34636346-090d5ffa-f26d-11e7-966e-4c1a1db3cdae 1](https://user-images.githubusercontent.com/58611/47627509-bd7a5d80-db06-11e8-8cc8-c4c9ff0562a9.png)


)

**Comment by dylanhs** [comment]
Can you add labels to the axes on that graph? Is that Bayesian Regret on the abscissa?

**Comment by endolith** [comment]
@dylanhs My graph?  The X axis is the distance from the winner to the center of the population.  I thin it might be more clear if it were symmetrical and X axis were just position of winner and candidates were uniformly distributed
